src/components/history/InfiniteList.js

```diff
@@ -1,17 +1,17 @@
 import PropTypes from 'prop-types'
 import React, {PureComponent} from 'react'
-import List from 'react-virtualized/dist/es/List'
-import AutoSizer from 'react-virtualized/dist/es/AutoSizer'
-import CellMeasurer from 'react-virtualized/dist/es/CellMeasurer'
+import AutoSizer from 'react-virtualized/dist/commonjs/AutoSizer'
```

Comment:
The tests broke since the new react-virtualized doesn't ship the babelrc anymore.

Reply:
Can we do on this way?
```
import { AutoSizer, CellMeasurer, List } from 'react-virtualized'
```

Reply:
We explicitly don't do it. Also for lodash, because this way we only import the files we need. In your proposal the bundle would include every file from react-virtualized even though we don't use all of them.
Ideally webpack can treeshake it, but I'm not sure we are on a late enough version.

Reply:
Got it, thanks for explanation